src/core/forms.py

```
@@ -11,6 +11,25 @@ class CoverImageWidget(forms.ClearableFileInput):
+    template_name = "widgets/star_rating.html"
+
+    def get_context(self, name, value, attrs):
+        """
+        Add score choices with their verbose names to the template context.
+
+        This allows the template to display the descriptive labels
+        (e.g., "Adored", "Loved") when hovering over stars.
+        """
+        context = super().get_context(name, value, attrs)
+        # Get the choices from the Media model's score field
+        score_field = Media._meta.get_field("score")  # noqa: SLF001
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)